core/src/com/unciv/models/ruleset/unique/Conditionals.kt

```diff
@@ -142,6 +142,8 @@ object Conditionals {
             UniqueType.ConditionalVictoryDisabled -> checkOnGameInfo { !gameParameters.victoryTypes.contains(conditional.params[0]) }
             UniqueType.ConditionalReligionEnabled -> checkOnGameInfo { isReligionEnabled() }
             UniqueType.ConditionalReligionDisabled -> checkOnGameInfo { !isReligionEnabled() }
+            UniqueType.ConditionalEspionageEnabled -> checkOnGameInfo { isEspionageEnabled() }
+            UniqueType.ConditionalEspionageDisabled -> checkOnGameInfo { !isEspionageEnabled() }
```

Comment:
Should be added to isUnavailableBySettings list of "disabled by game settings" uniques

Reply:
This is what makes it unavailable in civilopedia btw